variables.tf

```diff
@@ -322,4 +322,10 @@ variable "oidc_provider" {
   description = "EKS cluster OIDC provider name, required for configuring IAM using IRSA."
   type        = string
   default     = ""
-}
\ No newline at end of file
+}
+
+variable "queryFunctionsAcrossAllMetastores" {
```

Comment:
variable should be named as: `enable_query_functions_across_all_metastores` following the convention in this file.